### PR TITLE
Feature/adversary intelligence documentation

### PR DESCRIPTION
**Pull request by gtheodorio**
Created documentation for CrowdStrike Falcon Adversary Intelligence data connector.

### CODE REVIEW

File: adversary-intelligence/README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+## Technical Data
+
+This data connector imports Falcon IOCs into the Threat Intelligence upload indicators API on Microsoft Sentinel. The data connector is deployed as an Azure Function using an ARM template.
+
+### Data Flow
+
+![image](./media/dataconnector_diagram.png)
+
+- The function checks if there is an existing "_marker" in the file share. Using the marker, it queries the CrowdStrike Intel endpoint `/intel/combined/indicators/v1` for indicators based on the indicators type provided in the configuration. If there is no marker present, this means that it's the first execution, at which point an date will be used to retrieve the indicators based on the update date. This date is derived by the look back days value provided in the configuration.
```
Comment (carlosmmatos) [inline adversary-intelligence/README.md:11]:
```suggestion
- The function checks if there is an existing "_marker" in the file share. Using the marker, it queries the CrowdStrike Intel endpoint `/intel/combined/indicators/v1` for indicators based on the indicators type provided in the configuration. If there is no marker present, this means that it's the first execution, at which point a date will be used to retrieve the indicators based on the updated date. This date is derived by the `LookBackDays` value provided in the configuration.
```

File: adversary-intelligence/README.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+- Once all indicators have been mapped, they're uploaded to Sentinel Threat Intelligence using the [Threat Intel upload indicators API](https://learn.microsoft.com/en-us/azure/sentinel/upload-indicators-api).
+
+### Configuration
+
+The following parameters are required for the ARM template top be deployed.
+
+- FunctionName - A name to be given to the Azure Function App. Default is 'CSFalconIOC'
+- CrowdStrikeClientId - The client id of the credentials generated in the CrowdStrike console
+- CrowdStrikeClientSecret - The client secret of the credentials generated in the CrowdStrike console
+- CrowdStrikeBaseUrl - The base url of the region in which the CrowdStrike credentials were generated
```
Comment (carlosmmatos) [inline adversary-intelligence/README.md:64]: Not sure if this is something we control or not, but I would stick with our common naming schema:
FalconClientID

**Review by carlosmmatos — CHANGES_REQUESTED** [review]
See comments

File: adversary-intelligence/README.md (hunk 2 above)
Comment (gtheodorio) [reply adversary-intelligence/README.md:64]: Oof. We could, but the process to get that changed could take a while as the integration is already deployed to the marketplace.

**Review by carlosmmatos — APPROVED** [review]

**Comment by carlosmmatos** [comment]
we can deal with the spelling check stuff later.. it's really not that useful

**Pull request merged**